docs/build-parachains-rococo.md

```diff
@@ -0,0 +1,108 @@
+builders for the time being. It's recommended at this point to run your own parachains testnet
+locally.
+
+## How to connect to a Parachains
+
+If you would like to connect to a parachain via [Polkadot-JS Apps](https://polkadot.js.org/apps/),
+you may do so by clicking on the network selection at the top left hand corner of the navigation and
+selecting any parachain of choice. For the purpose of these following examples, we will be using the
+Rococo testnet "Custom Node" underneath "Development", according to the
+[parachain workshop](https://substrate.dev/cumulus-workshop/#/1-prep/1-compiling).
+
+![parachains on polkadotjs](assets/polkadotjs_network_parachains.png)
+
+## How to make Cross Chain transfers
+
+To send a transfer between parachains, navigate to "Accounts" > "Transfer". From here, you'll need
+to select the parachain node that you are running. Next, enter in the amount that you'd like to send
+to another parachain. Be sure to select the correct parachain you'd like to send an amount to. Once
+you've hit the "Submit" button, you should see a green notification, indicating a successful
+transfer.
+
+### Downward Transfers
+
+Downward transfers are when an account on the Relay Chain sends a transfer to their account on a
+different parachain. This type of transfer uses a depository and mint model, meaning that when the
+DOT leave the sender's account on the Relay Chain and are transferred into an account on a
+parachain, the parachain mints a corresponding amount of tokens on the parachain.
+
+For example, we can send tokens from Alice's account on the Relay Chain to her account on
+parachain 200. To do so, we will need to head to the "Network" > "Parachains" tab and click on the
+"Transfer to chain" button.
+
+![rococo downward transfer](assets/rococo/rococo-downward-transfer.png)
+
+Notice here, that we can select which parachain to send the funds to, specify the amount to be sent,
+and add any comments or a memo for the transfer.
+
+### Upward Transfers
+
+Upward transfers occur _from_ a parachain _to_ an account on the Relay Chain. To proceed with this
+kind of transfer, we need to be connected to a parachain node on the network and be on the
+"Network" > "Parachains" tab. Click on the "Transfer to chain" button.
+
+![rococo upward transfer](assets/rococo/rococo-upward-transfer.png)
+
+Note that the toggle should be set to off, ensuring that the funds go to the Relay Chain and not
+another parachain.
+
+### Lateral Transfers
+
+This type of transfer is only possible with at least two different registered parachains. In true
+XCMP, lateral transfers would allow for messages to be sent directly from one parachain to another.
+However, this is not yet implemented, so the Relay Chain is helping us deliver messages for the time
+being. Lateral transfers work through the depository model, which means that in order to transfer
+tokens from chain 200 to chain 300, there must already be tokens owned by chain 200 deposited on
```

Comment:
Wait, why? If I'm transferring X from A to B, why should B already have to have some of X?

Reply:
Yeah lateral transfers are confusing and I find that they don't make the most sense, in this [link](https://substrate.dev/cumulus-workshop/#/4-cross-chain/3-lateral?id=depository-model), it says that they might revisit this model in the future and explains more of why this model is the way that it is.

Reply:
Yeah that's silly. Maybe link to that here as well.